seblaifa3315/fun-kickoff-project
# profile setting skeleton

###  What this PR does (required):
I created the skeleton for the profile settings page (Side and main content).
I didn't style the inside of any of the main content, only the sidebar and the outside of the content. Also the the Navbar is not styled neither, it is there just to see the height of the main page.

###  See below three Screenshots as example:
Edit Profile View:
![edit profile view](https://user-images.githubusercontent.com/77294017/150653776-b052da9d-acaf-4464-87c0-40bcfe331dc1.jpg)

Profile Photo View
![profile photo view](https://user-images.githubusercontent.com/77294017/150653797-f25eed0d-a702-40ea-8f53-8e0e783c4a6e.jpg)

Availibility View:
![availability view](https://user-images.githubusercontent.com/77294017/150653812-6fc0318b-c5aa-4682-ae13-bda0d94c98b7.jpg)


### Any information needed to test this feature (required):
- Click on the links on the sidebar, it should switch views on the main content.
- The links should be bolder when you hover them or when it is active (when you are the current main content).
- Try to change the size of the screen and it should be responsive.
-
### Any issues with the current functionality (optional):
- I'm not sure if the font-size, font-family and spacing are  correct.
